src/widgets/bitcoin/BitcoinWidgetTile.tsx

```diff
@@ -0,0 +1,95 @@
+import { Badge, Center, Flex, Image, Text } from '@mantine/core';
+import { IconCurrencyBitcoin } from '@tabler/icons-react';
+import { useTranslation } from 'react-i18next';
+import { api } from '~/utils/api';
+
+import { defineWidget } from '../helper';
+import { WidgetLoading } from '../loading';
+import { IWidget } from '../widgets';
+
+const definition = defineWidget({
+  id: 'bitcoin',
+  icon: IconCurrencyBitcoin,
+  options: {},
```

Comment:
You should have the option to select the coin

Reply:
I didn't know what this options were all about therefore I left it blank